apps/music/js/ui/views/list_view.js

```diff
@@ -367,31 +414,22 @@ var ListView = {
       self.searchInput.value = '';
       SearchView.clearSearch();
     }
+
     var target = evt.target;
     if (!target) {
       return;
     }
 
     switch (evt.type) {
-      case 'touchend':
-        // Check for tap on parent form element with event origin as clear buton
-        // This is workaround for a bug in input_areas BB. See Bug 920770
-        if (target.id === 'views-list-search') {
-          var id = evt.originalTarget.id;
-          if (id && id !== 'views-list-search-input' &&
-            id !== 'views-list-search-close') {
-            lv_resetSearch(this);
-            return;
-          }
-        }
+      case 'contextmenu':
+        var option = target.dataset.option;
 
-        if (target.id === 'views-list-search-clear') {
-          lv_resetSearch(this);
-          return;
+        if (option === 'title') {
+          var playlistName = prompt("Add to playlist?");
```

Comment:
@hfiguiere  I actually want to change this to be the way people wanted it so it asks to pick bluetooth or add to playlist, but I didn't know how to do that and also provide a list of playlists people can choose from instead of just text input!